Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -73,31 +73,29 @@ public async Task LoadAll()
                 string countryCode = resource.Substring(RatingsResource.Length, 2);
                 var dict = new Dictionary<string, ParentalRating>(StringComparer.OrdinalIgnoreCase);
 
-                using (var str = _assembly.GetManifestResourceStream(resource))
-                using (var reader = new StreamReader(str))
+                await using var str = _assembly.GetManifestResourceStream(resource);
```

Comment:
Still need to call `.configureAwait(false)` for `await using`. Unfortunately the syntax to do so isn't very nice :/

```suggestion
                var str = _assembly.GetManifestResourceStream(resource);
                await using var asyncDisposableStr = str.configureAwait(false)
```